### PR TITLE
Amend/update profile

### PR DESCRIPTION
**Pull request by needToRoll**
This closes none
Summary of changes:
- ​I've updated my profile, finally
- @jmesserli could you please check my spelling and grammer once again (german and eglish please)
I made sure to:
- [ ] ~Add and / or update tests~
- [x] Run the code formatter (`CTRL + ALT + L`)
- [x] Update any documentation (javadoc, domain model, erd, ...)
- [x] Follow the contribution guidelines
- [ ] ~Update CI script and configs on Jenkins (`Jenkinsfile`, `auth0.yml`, `application.yml`, etc.)~

Please review @jmesserli  / cc: @outcobra/developers.

### CODE REVIEW

File: frontend/src/assets/docs/profiles.en.md [hunk 1]
```diff
@@ -20,16 +20,16 @@ If I am neither holding a mouse nor my camera, I grab my Vic Firth Sticks and ta
...
+Kotlin enthusiast, caffeine-junkie and sailor
 
-Thanks to the fact that I could use it in school, because of my physical disability, I rapidly spent a lot of time with it.
+My main focus in Outcobra is on the application logic.
+This means that I am responsible for the fact that the data you are typing is stored in the correct format, and will be available to you in the future on every device.
 
-Five years later I began to teach programming myself.
+Designing software never gets boring. If these work, at the end of the day, the feeling is indescribable.
+For this reason, and to keep improving myself, I spend a lot of my spare time with software development.
+On my desk, besides the mouse and the keyboard, a cup of coffee can not be missed.
```
Comment (jmesserli) [inline frontend/src/assets/docs/profiles.en.md]: Besides the mouse and the keyboard, a cup of coffee can't be missed on my desk.

Sounds better IMO

File: frontend/src/assets/docs/profiles.en.md [hunk 2]
```diff
@@ -20,16 +20,16 @@ If I am neither holding a mouse nor my camera, I grab my Vic Firth Sticks and ta
...
 
-The first contact with Computers I had in third grade. At that time I got my first own computer.
+Kotlin enthusiast, caffeine-junkie and sailor
 
-Thanks to the fact that I could use it in school, because of my physical disability, I rapidly spent a lot of time with it.
+My main focus in Outcobra is on the application logic.
+This means that I am responsible for the fact that the data you are typing is stored in the correct format, and will be available to you in the future on every device.
 
-Five years later I began to teach programming myself.
+Designing software never gets boring. If these work, at the end of the day, the feeling is indescribable.
```
Comment (jmesserli) [inline frontend/src/assets/docs/profiles.en.md]: If **the software** works at the end of the day, the feeling is indescribable.

File: frontend/src/app/exam/exam.component.ts [hunk 3]
```diff
@@ -144,6 +144,7 @@ export class ExamComponent implements OnInit, AfterViewInit {
                 this._examService.readById(incompleteExam.id)
                     .subscribe((completeExam: ExamDto) => {
                         this._allExams.push(completeExam);
+                        this._displayForFilter();
```
Comment (jmesserli) [inline frontend/src/app/exam/exam.component.ts]: ?

Comment (M4R1KU) [reply frontend/src/app/exam/exam.component.ts]: ?

File: frontend/src/assets/docs/profiles.de.md [hunk 4]
```diff
@@ -11,13 +11,16 @@ Try-Hard UX/UI Fanatiker, Hobbyfotograf und fast professioneller Drummer
...
 ## Florian
-Den ersten Kontakt zur Informatik hatte ich der dritten Klasse... damals erhielt ich meinen ersten Computer.
-Da ich diesen als Nachteilsausgleich in der Schule verwenden durfte, verbrachte ich schnell viel Zeit damit.
-5 Jahre später habe ich begonnen mir selber das Programmieren beizubringen.
-Heute mache ich eine Ausbildung als Softwareentwickler und beschäftige mich zu Bürozeiten hauptsächlich mit der Qualitätssicherung von Softwareprodukten. Um Entwicklungstechnisch auf dem neusten Stand zu blieben, realisiere ich immer wieder kleinere Nebenprojekte.
-Outcobra ist das bisher grösste dieser "Nebenprojekte". Meine Hauptaufgaben liegen hier in der Backend-Entwicklung und im Testing-Bereich.
-Für mich ist es der Mix aus Bekanntem und vielen neuen Herausforderungen der mich an diesem Projekt fasziniert!
+Kotlin enthusiast, Koffein-Junkie und Segler
 
+Mein Aufgabenbereich in Outcobra liegt hauptsächlich in der Applikationslogik.
```
Comment (M4R1KU) [inline frontend/src/assets/docs/profiles.de.md]: Etwas liegt *auf* der Logik oder mein Aufgabenbereich befindet sich

File: frontend/src/assets/docs/profiles.de.md [hunk 5]
```diff
@@ -11,13 +11,16 @@ Try-Hard UX/UI Fanatiker, Hobbyfotograf und fast professioneller Drummer
...
-5 Jahre später habe ich begonnen mir selber das Programmieren beizubringen.
-Heute mache ich eine Ausbildung als Softwareentwickler und beschäftige mich zu Bürozeiten hauptsächlich mit der Qualitätssicherung von Softwareprodukten. Um Entwicklungstechnisch auf dem neusten Stand zu blieben, realisiere ich immer wieder kleinere Nebenprojekte.
-Outcobra ist das bisher grösste dieser "Nebenprojekte". Meine Hauptaufgaben liegen hier in der Backend-Entwicklung und im Testing-Bereich.
-Für mich ist es der Mix aus Bekanntem und vielen neuen Herausforderungen der mich an diesem Projekt fasziniert!
+Kotlin enthusiast, Koffein-Junkie und Segler
 
+Mein Aufgabenbereich in Outcobra liegt hauptsächlich in der Applikationslogik. 
+Das bedeutet, ich bin verantwortlich dafür, dass die Daten die Du eintippst auch im richtigen Format gespeichert werden, und dir in Zukunft auf jedem Gerät zur Verfügung stehen. 
+
+Das entwerfen von Programmen wird mir nie langweilig. Wenn diese dann auch noch funktionieren ist die Freude unbeschreiblich.
```
[the remark this answers is not in the document]
Comment (M4R1KU) [reply frontend/src/assets/docs/profiles.de.md]: De satz eif imo komisch u es tönt aus wär ds Programmiere glückssach.

File: frontend/src/assets/docs/profiles.de.md [hunk 6]
```diff
@@ -11,13 +11,16 @@ Try-Hard UX/UI Fanatiker, Hobbyfotograf und fast professioneller Drummer
...
-Heute mache ich eine Ausbildung als Softwareentwickler und beschäftige mich zu Bürozeiten hauptsächlich mit der Qualitätssicherung von Softwareprodukten. Um Entwicklungstechnisch auf dem neusten Stand zu blieben, realisiere ich immer wieder kleinere Nebenprojekte.
-Outcobra ist das bisher grösste dieser "Nebenprojekte". Meine Hauptaufgaben liegen hier in der Backend-Entwicklung und im Testing-Bereich.
-Für mich ist es der Mix aus Bekanntem und vielen neuen Herausforderungen der mich an diesem Projekt fasziniert!
+Kotlin enthusiast, Koffein-Junkie und Segler
 
+Mein Aufgabenbereich in Outcobra liegt hauptsächlich in der Applikationslogik. 
+Das bedeutet, ich bin verantwortlich dafür, dass die Daten die Du eintippst auch im richtigen Format gespeichert werden, und dir in Zukunft auf jedem Gerät zur Verfügung stehen. 
+
+Das entwerfen von Programmen wird mir nie langweilig. Wenn diese dann auch noch funktionieren ist die Freude unbeschreiblich. 
+Aus diesem Grund, und um mich selber sehts zu verbessern, verbringe ich einen grossen Teil meiner Freizeit mit der Softwareentwicklung.
```
Comment (M4R1KU) [inline frontend/src/assets/docs/profiles.de.md]: sehts -> stets

File: frontend/src/assets/docs/profiles.de.md [hunk 7]
```diff
@@ -11,13 +11,16 @@ Try-Hard UX/UI Fanatiker, Hobbyfotograf und fast professioneller Drummer
...
+Kotlin enthusiast, Koffein-Junkie und Segler
 
+Mein Aufgabenbereich in Outcobra liegt hauptsächlich in der Applikationslogik. 
+Das bedeutet, ich bin verantwortlich dafür, dass die Daten die Du eintippst auch im richtigen Format gespeichert werden, und dir in Zukunft auf jedem Gerät zur Verfügung stehen. 
+
+Das entwerfen von Programmen wird mir nie langweilig. Wenn diese dann auch noch funktionieren ist die Freude unbeschreiblich. 
+Aus diesem Grund, und um mich selber sehts zu verbessern, verbringe ich einen grossen Teil meiner Freizeit mit der Softwareentwicklung. 
+Auf meinem Schreibtsch darf dabei nebst Maus und Tastatur auch die Tasse Kaffee nicht fehlen.
+
+Als ausgleich zur Informatik geniesse ich die Ruhe auf dem See. Damit mir auch dort nicht langweilig wird, habe ich vor einigen Jahren begonnen, an Regattas (Segelwettbewerben) teil zu nehmen.
```
[the remark this answers is not in the document]
Comment (M4R1KU) [reply frontend/src/assets/docs/profiles.de.md]: teilzunehmen

File: frontend/src/assets/docs/profiles.en.md [hunk 8]
```diff
@@ -20,16 +20,16 @@ If I am neither holding a mouse nor my camera, I grab my Vic Firth Sticks and ta
...
+For this reason, and to keep improving myself, I spend a lot of my spare time with software development.
+On my desk, besides the mouse and the keyboard, a cup of coffee can not be missed.
 
-Today I am in an apprenticeship as software developer with a focus on Quality Assurance. To be up to date with the current trends in development I realize many side projects.
-Outcobra is to date my biggest one. I mainly focus on backend development and testing.
-
-What fascinates me about the project is the challenge of many known and unknown things which confront me every time.
+As a compensation to my activities in computer science, I enjoy the peace on the lake. To make sure I don't get bored, I started taking part in regattas (sailing contests) a few years ago.
```
Comment (M4R1KU) [inline frontend/src/assets/docs/profiles.en.md]: the calm on the lake odr gnessisch dr letzt friede?

**Pull request merged**